apis/cli/v1alpha1/cliplugin_types.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ArtifactList contains an Artifact object for every supported platform of a version.
+type ArtifactList []Artifact
+
+// OCIImage is a fully qualified OCI image of the plugin binary.
+type OCIImage string
```

Comment:
Same here. Why not just a string? Do we have plans to define constants?

Reply:
Ack. Updated